resources/src/books.php

```diff
@@ -38,9 +37,9 @@ public function search ($name)
          $stmt2 = $this->pdo->query('SELECT * FROM Book where title LIKE "%'.$name.'%"');
          $op=array();
          $row=0;
-         if ($stmt2->fetch(PDO::FETCH_ASSOC))
```

Comment:
Please verify once PDO::FETCH_ASSOC, You might need to pass this into fetch method.